packages/permissions/src/types/generated/mc.ts

```diff
@@ -312,6 +312,7 @@ export enum TPermissionScope {
   GetPermissionForAnyProject = 'get_permission_for_any_project',
   IntrospectOauthTokens = 'introspect_oauth_tokens',
   ManageApiClients = 'manage_api_clients',
+  ManageAssociateRoles = 'manage_associate_roles',
```

Comment:
Were these included when the type generation script ran or do they relate to the `createdAt` work? Just curious

Reply:
Apparently so!